dessia_common/datatools/cluster.py

```diff
@@ -315,12 +315,13 @@ def plot_data(self):
         return Dataset.plot_data(self)
 
     def _plot_data_list(self):
-        _plot_data_list = []
-        for row, label in enumerate(self.labels):
-            _plot_data_list.append({attr: self.matrix[row][col] for col, attr in enumerate(self.common_attributes)})
-            _plot_data_list[-1]["Cluster Label"] = label
-            # (label if label != -1 else "Excluded") plot_data "Excluded" -> NaN
-        return _plot_data_list
+        plot_data_list = []
```

Comment:
`samples` would be a pretty neat name for this in my opinion (follows conventions, is consistent with sample_values defined afterwards, is straight to the point)

Reply:
Done